stravalib/client.py

```diff
@@ -263,12 +294,8 @@ def get_activities(self, before=None, after=None, limit=None):
 
         """
 
-        if before:
-            before = self._utc_datetime_to_epoch(before)
-
-        if after:
-            after = self._utc_datetime_to_epoch(after)
-
+        before_epoch = self._utc_datetime_to_epoch(before) if before else None
+        after_epoch = self._utc_datetime_to_epoch(after) if after else None
         params = dict(before=before, after=after)
```

Comment:
I don't see where `before_epoch` or `after_epoc` are used. Can it be that these were meant to be set as part of the `params` dict?

Reply:
Could catch they should be passed as param. I'll fix that